### PR TITLE
Carla pip dependency and fix for git init

### PR DESCRIPTION
**Pull request by gijskoning**
Running ./bridge.py fails on import Carla. This fixes it

### CODE REVIEW

**Comment by pd0wm** [comment]
@adeebshihadeh any reason you didn't do this in https://github.com/commaai/openpilot/commit/a3c26a912a9d26233d040bd71fe5324cd25142fd? Or just leftover when the carla in pip was broken?

**Comment by pd0wm** [comment]
[Maybe CI is failing just because te](https://github.com/commaai/openpilot/blob/master/tools/sim/Dockerfile.sim#L39-L41) can be removed.

**Comment by pd0wm** [comment]
Just rebuilding the docker container is probably causing CI to fail (git upgrade to new version?). Feel free to do small fixes in this PR to fix CI, if it becomes too much split it out into a separate PR.

**Comment by adeebshihadeh** [comment]
> @adeebshihadeh any reason you didn't do this in [a3c26a9](https://github.com/commaai/openpilot/commit/a3c26a912a9d26233d040bd71fe5324cd25142fd)? Or just leftover when the carla in pip was broken?

Don’t remember. It’s probably fine to add it.

File: Dockerfile.openpilot_base [hunk 1]
```diff
@@ -25,8 +25,9 @@ RUN cd /tmp && \
     rm -rf /tmp/* && \
     rm -rf /root/.cache && \
     pip uninstall -y pipenv && \
-
     # remove unused architectures from gcc for panda
     cd /usr/lib/gcc/arm-none-eabi/9.2.1 && \
     rm -rf arm/ && \
-    rm -rf thumb/nofp thumb/v6* thumb/v8* thumb/v7+fp thumb/v7-r+fp.sp
+    rm -rf thumb/nofp thumb/v6* thumb/v8* thumb/v7+fp thumb/v7-r+fp.sp && \
+    sudo git config --global --add safe.directory /tmp/openpilot
```
Comment (gijskoning) [inline Dockerfile.openpilot_base:32]: Hopefully a fix for https://github.blog/2022-04-12-git-security-vulnerability-announced/

**Pull request merged**